diff --git i/release.nix w/release.nix
index 1e685bf..fc0934e 100644
--- i/release.nix
+++ w/release.nix
@@ -1,9 +1,10 @@
 { nixopsVbox ? { outPath = ./.; revCount = 0; shortRev = "abcdef"; rev = "HEAD"; }
 , nixpkgs ? <nixpkgs>
 , officialRelease ? false
+, system ? builtins.currentSystem
 }:
 let
-  pkgs = import nixpkgs {};
+  pkgs = import nixpkgs { inherit system; };
   version =  "1.7" + (if officialRelease then "" else "pre${toString nixopsVbox.revCount}_${nixopsVbox.shortRev}");
 in
   rec {
